components/ArticleReplySummary.js

```diff
@@ -38,9 +43,23 @@ function ArticleReplySummary({ articleReply, className, ...props }) {
     </ProfileTooltip>
   );
 
+  useEffect(() => {
+    if (anchorRef.current.hash === location.hash) {
+      anchorRef.current.scrollIntoView({ behavior: 'smooth' });
+    }
+  }, []);
```

Comment:
Personally I prefer [scroll-behavior CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior) than imperative JS scrolling. I think it's OK to apply scroll-behavior globally